packages/atomic/src/components/facets/atomic-segmented-facet-scrollable/atomic-segmented-facet-scrollable.tsx

```diff
@@ -18,16 +43,21 @@ export class AtomicSegmentedFacetScrollable {
           style="square-neutral"
           class="flex shrink-0 basis-8 justify-center items-center rounded"
           ariaHidden="true"
+          onClick={() => this.scrollLeft.emit()}
         >
           <atomic-icon class="w-3.5" icon={ArrowLeftIcon}></atomic-icon>
         </Button>
-        <div class="wrapper-segmented flex flex-row overflow-x-scroll">
+        <div
+          id="horizontalScroll"
+          class="wrapper-segmented flex flex-row overflow-x-scroll scroll-smooth"
+        >
           <slot></slot>
         </div>
         <Button
           style="square-neutral"
           class="flex shrink-0 basis-8 justify-center items-center rounded"
           ariaHidden="true"
+          onClick={() => this.scrollRight.emit()}
```

Comment:
Using event's here would be a good solution if there's a child component which would emit the event too. Otherwise you can just call this.scroll(right/left)